examples/tutorial/tutorial_1_first_steps.ipynb

```diff
@@ -737,84 +737,79 @@
-      "The phone spn had 220 occurances and an average duration of 0.549409090909092\n",
-      "The phone D had 9128 occurances and an average duration of 0.062369631901840354\n",
-      "The phone IY1 had 3373 occurances and an average duration of 0.10762822413281947\n",
-      "The phone AW1 had 1007 occurances and an average duration of 0.16140019860973234\n"
+      "The phone <SIL> had 10956 occurrences and an average duration of 0.01746950301204816.\n",
+      "The phone AA0 had 66 occurrences and an average duration of 0.08999999999999986.\n",
+      "The phone AA1 had 8976 occurrences and an average duration of 0.1100735294117646.\n",
+      "The phone AA2 had 264 occurrences and an average duration of 0.06749999999999992.\n",
+      "The phone AE0 had 66 occurrences and an average duration of 0.10000000000000009.\n",
+      "The phone AE1 had 13200 occurrences and an average duration of 0.10714999999999965.\n",
+      "The phone AE2 had 594 occurrences and an average duration of 0.10333333333333337.\n",
+      "The phone AH0 had 43560 occurrences and an average duration of 0.058424242424242316.\n",
+      "The phone AH1 had 13662 occurrences and an average duration of 0.0728502415458933.\n",
+      "The phone AH2 had 198 occurrences and an average duration of 0.04666666666666669.\n",
+      "The phone AO0 had 198 occurrences and an average duration of 0.07333333333333328.\n",
+      "The phone AO1 had 8184 occurrences and an average duration of 0.11306451612903229.\n",
+      "The phone AO2 had 198 occurrences and an average duration of 0.0833333333333335.\n",
+      "The phone AW1 had 3366 occurrences and an average duration of 0.1503921568627447.\n",
+      "The phone AW2 had 66 occurrences and an average duration of 0.09999999999999998.\n",
+      "The phone AY0 had 66 occurrences and an average duration of 0.08000000000000007.\n",
+      "The phone AY1 had 12276 occurrences and an average duration of 0.16607526881720397.\n",
+      "The phone AY2 had 792 occurrences and an average duration of 0.10999999999999983.\n",
+      "The phone B had 11154 occurrences and an average duration of 0.0695266272189347.\n",
+      "The phone CH had 3036 occurrences and an average duration of 0.11260869565217396.\n",
+      "The phone D had 26928 occurrences and an average duration of 0.06436274509803894.\n",
+      "The phone DH had 19272 occurrences and an average duration of 0.05547945205479459.\n",
+      "The phone EH0 had 330 occurrences and an average duration of 0.06400000000000004.\n",
+      "The phone EH1 had 17556 occurrences and an average duration of 0.07954887218045083.\n",
+      "The phone ER0 had 10626 occurrences and an average duration of 0.08621118012422366.\n",
+      "The phone ER1 had 2772 occurrences and an average duration of 0.12023809523809531.\n",
+      "The phone EY1 had 9240 occurrences and an average duration of 0.1429285714285711.\n",
```

Comment:
similarly to the last comment: something is wrong here, judging by the counts.

There should be 92 occurrences of `ZH`, from executing this in `LibriSpeech-aligned`:
```
grep -r "text = \"ZH" . | wc
```
This is the count in the old output, but not the new output (